_includes/head.pug

```diff
@@ -37,3 +37,6 @@ head
 
   // Twitter cards
   | {% include twitter_cards.html %}
+
+  // Schema.org
+  | {% include schema-index.html %}
```

Comment:
Maybe we should do this, I could also do it in a later PR after testing it locally.

```suggestion
  | {% if page.url == "/" %}
  | {% include schema-index.html %}
  | {% endif %}
```